config/filter.d/murmur.conf

```diff
@@ -0,0 +1,21 @@
+# Fail2Ban filter for murmur/mumble-server
+#
+
+[INCLUDES]
+
+before = common.conf
+
+
+[Definition]
+
+_daemon = murmurd
+
+failregex = Rejected connection from <HOST>:\d+: Invalid server password$
+            Rejected connection from <HOST>:\d+: Wrong certificate or password for existing user$
```

Comment:
if works for you -- great -- add it in, add a ChangeLog and we will get it merged